canonical-web-and-design/jaas.ai
# Sitemap (including docs URLs) to submit to Google

Similar to https://github.com/canonical-web-and-design/maas.io/pull/370.

Submitting URLs to Google one at a time is just too cumbersome. So instead, we need to submit a sitemap.

Although [the help page](https://support.google.com/webmasters/answer/183668?hl=en) suggests that you can just upload a file:

> ... or directly submitting it to Search Console.

This doesn't appear to be the case in the actual search console:

![image](https://user-images.githubusercontent.com/519935/61944526-c6189b00-af95-11e9-9a24-ea57330ed098.png)

So I'm uploading this file, in txt format, to then submit the URL to Google, as described:

> If your sitemap includes only web page URLs, you can provide Google with a simple text file that contains one URL per line. For example:
> 
> ```
> http://www.example.com/file1.html
> http://www.example.com/file2.html
> ```
> 
> Guidelines for text file sitemaps:
> 
> - Encode your file using UTF-8 encoding.
> - Your text file should contain nothing but the list of URLs.
> - You can name the text file anything you wish, provided it has a .txt extension (for instance, sitemap.txt).

QA
--

Visit https://jaas-ai-canonical-web-and-design-pr-388.run.demo.haus/static/sitemap.txt - check it looks about right.
